test/dynamo/test_dynamic_shapes.py

```diff
@@ -31,6 +31,8 @@
     "ReproTests": [
         # Could not infer dtype of torch._C.SymIntNode
         "test_convert_boxes_to_pooler_format",
+        # TODO: look into this: FakeTensor error.
+        "test_gan_repro_trying_to_backward_through_the_graph_a_second_time",
```

Comment:
The error I hit was very weird, and seemed unrelated to the actual issue in this PR. Ignoring so we can land this PR asap, and I'll come back to it later.